pkg/universe.dagger.io/x/wangwei27494731@gmail.com/kustomize/kustomize.cue

```diff
@@ -0,0 +1,59 @@
+package kustomize
+
+import (
+	"universe.dagger.io/bash"
+	"universe.dagger.io/docker"
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+)
+
+// Apply a Kubernetes Kustomize folder
+#Kustomize: {
+	// Kubernetes source
+	source: string
+
+	// Optional Kustomization file
+	kustomization: string
+
+	// Kubeconfig
+	kubeconfig: dagger.#Secret
+
+	_baseImage: #Image
+
+	_file: core.#Source & {
+		path: source
+	}
```

Comment:
You should take a `dagger.#FS` instead of sourcing it with `core.#Source`.
The problem is that `core.#Source` is relative to the cue file himself, I think you would prefer taking a filesystem as input and apply kustomization on it